build.gradle

```diff
@@ -21,31 +21,14 @@ buildscript {
 
 plugins {
     id 'com.diffplug.spotless' version '6.25.0'
-    id 'eclipse'
     id 'com.github.ben-manes.versions' version '0.51.0'
-    id 'biz.aQute.bnd.builder' version '7.0.0'
-    id 'ru.vyarus.animalsniffer' version '1.7.1'
 }
 
-description = 'Mockito mock objects library core API and implementation'
+apply from: "$rootDir/gradle/shipkit.gradle"
+apply from: "$rootDir/gradle/root/ide.gradle"
+apply from: "$rootDir/gradle/license.gradle"
 
-apply plugin: 'base'
-base.archivesName = 'mockito-core'
-
-apply from: 'gradle/shipkit.gradle'
-
-apply from: 'gradle/root/ide.gradle'
-apply from: 'gradle/root/gradle-fix.gradle'
```

Comment:
I don't see this file in either the `rootProject` or in `mockito-core`. Does that mean it is unused now and should we delete it? It seems like it should be part of the `rootProject`, but looking at the contents of the file, I am not sure if it is still relevant on recent versions of Gradle.

Reply:
Yeah I didn't add the fix since it is not anymore necessary.
My plan is to drop this file in subsequent work on build scripts. I can do it now though.